install-php-extensions

```diff
@@ -2238,6 +2244,10 @@ installRemoteModule() {
 				fi
 			fi
 			;;
+		xmldiff)
+			# whether to use system diffmark library
+			addConfigureOption 'with-libdiffmark' 'yes'
```

Comment:
I don't see any `with-libdiffmark` option: see https://mlocati.github.io/pecl-info/#xmldiff

Reply:
See it on source code http://svn.php.net/viewvc/pecl/xmldiff/tags/release_1.1.3/config.m4?revision=352303&view=markup
Perharps I misunderstood

Reply:
For such options we have to look into the package.xml file (or use that pecl-info website)

Reply:
Don't worry about it, because I found that `libdiffmark` is not supported with PHP 8.x (so I've disabled it)